--- openldap-2.4.39/contrib/slapd-modules/smbk5pwd/Makefile.0003~	2014-02-16 12:51:03.575329721 +0100
+++ openldap-2.4.39/contrib/slapd-modules/smbk5pwd/Makefile	2014-02-16 13:26:34.507275902 +0100
@@ -28,9 +28,9 @@ LIBTOOL = $(LDAP_BUILD)/libtool
 CC = gcc
 OPT = -g -O2 -Wall
 # Omit DO_KRB5, DO_SAMBA or DO_SHADOW if you don't want to support it.
-DEFS = -DDO_KRB5 -DDO_SAMBA -DDO_SHADOW
+DEFS = -DDO_SAMBA -DDO_SHADOW
 INCS = $(LDAP_INC) $(HEIMDAL_INC) $(SSL_INC)
-LIBS = $(LDAP_LIB) $(HEIMDAL_LIB) $(SSL_LIB)
+LIBS = $(LDAP_LIB) $(SSL_LIB)
 
 PROGRAMS = smbk5pwd.la
 LTVER = 0:0:0
