--- openldap-2.4.39/contrib/slapd-modules/smbk5pwd/Makefile.0002~	2014-02-16 12:49:36.944331909 +0100
+++ openldap-2.4.39/contrib/slapd-modules/smbk5pwd/Makefile	2014-02-16 12:50:31.892330522 +0100
@@ -21,8 +21,8 @@ LDAP_LIB = $(LDAP_BUILD)/libraries/libld
 SSL_INC = 
 SSL_LIB = -lcrypto
 
-HEIMDAL_INC = -I/usr/heimdal/include
-HEIMDAL_LIB = -L/usr/heimdal/lib -lkrb5 -lkadm5srv
+HEIMDAL_INC = -I/usr/include/heimdal
+HEIMDAL_LIB = -L$(libdir)/heimdal -lkrb5 -lkadm5srv
 
 LIBTOOL = $(LDAP_BUILD)/libtool
 CC = gcc
