diff -uraN openldap-2.5.6/libraries/liblber/Makefile.in omv-openldap-2.5.6/libraries/liblber/Makefile.in
--- openldap-2.5.6/libraries/liblber/Makefile.in	2021-07-27 19:44:47.000000000 +0200
+++ omv-openldap-2.5.6/libraries/liblber/Makefile.in	2021-07-28 00:32:31.798505609 +0200
@@ -51,6 +51,6 @@
 
 install-local: FORCE
 	-$(MKDIR) $(DESTDIR)$(libdir)
-	$(LTINSTALL) $(INSTALLFLAGS) -m 644 $(LIBRARY) $(DESTDIR)$(libdir)
+	$(LTINSTALL) $(INSTALLFLAGS) -m 755 $(LIBRARY) $(DESTDIR)$(libdir)
 	$(LTFINISH) $(DESTDIR)$(libdir)
 
diff -uraN openldap-2.5.6/libraries/libldap/Makefile.in omv-openldap-2.5.6/libraries/libldap/Makefile.in
--- openldap-2.5.6/libraries/libldap/Makefile.in	2021-07-27 19:44:47.000000000 +0200
+++ omv-openldap-2.5.6/libraries/libldap/Makefile.in	2021-07-28 00:33:35.610508269 +0200
@@ -82,7 +82,7 @@
 
 install-local: $(CFFILES) FORCE
 	-$(MKDIR) $(DESTDIR)$(libdir)
-	$(LTINSTALL) $(INSTALLFLAGS) -m 644 $(LIBRARY) $(DESTDIR)$(libdir)
+	$(LTINSTALL) $(INSTALLFLAGS) -m 755 $(LIBRARY) $(DESTDIR)$(libdir)
 	$(LTFINISH) $(DESTDIR)$(libdir)
 	-$(MKDIR) $(DESTDIR)$(sysconfdir)
 	@for i in $(CFFILES); do \
