diff -Naur --exclude '*~' openldap-2.4.8/libraries/liblber/Makefile.in openldap-2.4.8-fix-library-perms/libraries/liblber/Makefile.in
--- openldap-2.4.8/libraries/liblber/Makefile.in	2008-02-11 18:26:41.000000000 -0500
+++ openldap-2.4.8-fix-library-perms/libraries/liblber/Makefile.in	2008-07-23 09:27:55.000000000 -0400
@@ -48,6 +48,6 @@
 
 install-local: FORCE
 	-$(MKDIR) $(DESTDIR)$(libdir)
-	$(LTINSTALL) $(INSTALLFLAGS) -m 644 $(LIBRARY) $(DESTDIR)$(libdir)
+	$(LTINSTALL) $(INSTALLFLAGS) -m 755 $(LIBRARY) $(DESTDIR)$(libdir)
 	$(LTFINISH) $(DESTDIR)$(libdir)
 
diff -Naur --exclude '*~' openldap-2.4.8/libraries/libldap/Makefile.in openldap-2.4.8-fix-library-perms/libraries/libldap/Makefile.in
--- openldap-2.4.8/libraries/libldap/Makefile.in	2008-02-11 18:26:41.000000000 -0500
+++ openldap-2.4.8-fix-library-perms/libraries/libldap/Makefile.in	2008-07-23 09:28:20.000000000 -0400
@@ -64,7 +64,7 @@
 
 install-local: $(CFFILES) FORCE
 	-$(MKDIR) $(DESTDIR)$(libdir)
-	$(LTINSTALL) $(INSTALLFLAGS) -m 644 $(LIBRARY) $(DESTDIR)$(libdir)
+	$(LTINSTALL) $(INSTALLFLAGS) -m 755 $(LIBRARY) $(DESTDIR)$(libdir)
 	$(LTFINISH) $(DESTDIR)$(libdir)
 	-$(MKDIR) $(DESTDIR)$(sysconfdir)
 	@for i in $(CFFILES); do \
diff -Naur --exclude '*~' openldap-2.4.8/libraries/libldap_r/Makefile.in openldap-2.4.8-fix-library-perms/libraries/libldap_r/Makefile.in
--- openldap-2.4.8/libraries/libldap_r/Makefile.in	2008-02-11 18:26:41.000000000 -0500
+++ openldap-2.4.8-fix-library-perms/libraries/libldap_r/Makefile.in	2008-07-23 09:28:36.000000000 -0400
@@ -79,6 +79,6 @@
 
 install-local: $(CFFILES) FORCE
 	-$(MKDIR) $(DESTDIR)$(libdir)
-	$(LTINSTALL) $(INSTALLFLAGS) -m 644 $(LIBRARY) $(DESTDIR)$(libdir)
+	$(LTINSTALL) $(INSTALLFLAGS) -m 755 $(LIBRARY) $(DESTDIR)$(libdir)
 	$(LTFINISH) $(DESTDIR)$(libdir)
 
