diff -uraN openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile omv-openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile
--- openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile	2021-07-27 19:44:47.000000000 +0200
+++ omv-openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile	2021-07-28 00:23:14.105482366 +0200
@@ -29,9 +29,9 @@
 CC = gcc
 OPT = -g -O2
 # Omit DO_KRB5, DO_SAMBA or DO_SHADOW if you don't want to support it.
-DEFS = -DDO_KRB5 -DDO_SAMBA -DDO_SHADOW
+DEFS = -DDO_SAMBA -DDO_SHADOW
 INCS = $(LDAP_INC) $(HEIMDAL_INC) $(SSL_INC)
-LIBS = $(LDAP_LIB) $(HEIMDAL_LIB) $(SSL_LIB)
+LIBS = $(LDAP_LIB) $(SSL_LIB)
 
 PROGRAMS = smbk5pwd.la
 MANPAGES = slapo-smbk5pwd.5
