diff -uraN openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile omv-openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile
--- openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile	2021-07-27 19:44:47.000000000 +0200
+++ omv-openldap-2.5.6/contrib/slapd-modules/smbk5pwd/Makefile	2021-07-28 00:11:48.633453796 +0200
@@ -21,8 +21,8 @@
 SSL_INC = 
 SSL_LIB = -lcrypto
 
-HEIMDAL_INC = -I/usr/heimdal/include
-HEIMDAL_LIB = -L/usr/heimdal/lib -lkrb5 -lkadm5srv
+HEIMDAL_INC = -I/usr/include/heimdal
+HEIMDAL_LIB = -L$(libdir)/heimdal -lkrb5 -lkadm5srv
 
 LIBTOOL = $(LDAP_BUILD)/libtool
 INSTALL = /usr/bin/install
