diff -up openldap-2.6.10/configure.ac.13~ openldap-2.6.10/configure.ac
--- openldap-2.6.10/configure.ac.13~	2026-01-11 20:08:55.169419786 +0000
+++ openldap-2.6.10/configure.ac	2026-01-11 20:08:55.149419742 +0000
@@ -821,19 +821,19 @@ dnl Check for module support
 ol_link_modules=no
 WITH_MODULES_ENABLED=no
 if test $ol_enable_modules != no ; then
-	AC_CHECK_HEADERS(ltdl.h)
+	AC_CHECK_HEADERS(sltdl/sltdl.h)
 
-	if test $ac_cv_header_ltdl_h = no ; then
-		AC_MSG_ERROR([could not locate libtool ltdl.h])
+	if test $ac_cv_header_sltdl_sltdl_h = no ; then
+		AC_MSG_ERROR([could not locate sltdl.h])
 	fi
 
-	AC_CHECK_LIB(ltdl, lt_dlinit, [
-		MODULES_LIBS=-lltdl
-		AC_DEFINE(HAVE_LIBLTDL,1,[define if you have libtool -ltdl])
+	AC_CHECK_LIB(sltdl, lt_dlinit, [
+		MODULES_LIBS=-lsltdl
+		AC_DEFINE(HAVE_LIBLTDL,1,[define if you have -lsltdl])
 	])
 
-	if test "$ac_cv_lib_ltdl_lt_dlinit" = no ; then
-		AC_MSG_ERROR([could not locate libtool -lltdl])
+	if test "$ac_cv_lib_lstdl_lt_dlinit" = no ; then
+		AC_MSG_ERROR([could not locate libtool -lsltdl])
 	fi
 	ol_link_modules=yes
 	WITH_MODULES_ENABLED=yes
@@ -2491,16 +2491,16 @@ dnl Sort out defines
 if test "$ol_enable_slapi" != no ; then
 	dnl This check is done also if --enable-modules is used;
 	dnl it is duplicated here, 'cause it'd be cached anyway
-	AC_CHECK_HEADERS(ltdl.h)
+	AC_CHECK_HEADERS(sltdl/sltdl.h)
 
-	if test $ac_cv_header_ltdl_h != yes ; then
-		AC_MSG_ERROR([could not locate <ltdl.h>])
+	if test $ac_cv_header_sltdl_sltdl_h != yes ; then
+		AC_MSG_ERROR([could not locate <sltdl/sltdl.h>])
 	fi
-	AC_CHECK_LIB(ltdl, lt_dlinit, [
-		SLAPI_LIBS=-lltdl
+	AC_CHECK_LIB(sltdl, lt_dlinit, [
+		SLAPI_LIBS=-lsltdl
 		LIBSLAPI=slapi/libslapi.la
-		AC_DEFINE(HAVE_LIBLTDL,1,[define if you have libtool -ltdl])
-	],[AC_MSG_ERROR([could not locate libtool -lltdl])])
+		AC_DEFINE(HAVE_LIBLTDL,1,[define if you have -lsltdl])
+	],[AC_MSG_ERROR([could not locate -lsltdl])])
 
 	AC_DEFINE(LDAP_SLAPI,1, [define this to add SLAPI code])
 fi
diff -up openldap-2.6.10/servers/slapd/module.c.13~ openldap-2.6.10/servers/slapd/module.c
--- openldap-2.6.10/servers/slapd/module.c.13~	2026-01-11 20:08:39.749385670 +0000
+++ openldap-2.6.10/servers/slapd/module.c	2026-01-11 20:47:33.467054850 +0000
@@ -19,7 +19,7 @@
 
 #ifdef SLAPD_MODULES
 
-#include <ltdl.h>
+#include <sltdl/sltdl.h>
 
 typedef int (*MODULE_INIT_FN)(
 	int argc,
@@ -119,16 +119,7 @@ int module_unload( const char *file_name
 
 static lt_dlhandle slapd_lt_dlopenext_global( const char *filename )
 {
-	lt_dlhandle handle = 0;
-	lt_dladvise advise;
-
-	if (!lt_dladvise_init (&advise) && !lt_dladvise_ext (&advise)
-			&& !lt_dladvise_global (&advise))
-		handle = lt_dlopenadvise (filename, advise);
-
-	lt_dladvise_destroy (&advise);
-
-	return handle;
+	return lt_dlopenext (filename);
 }
 
 int module_load(const char* file_name, int argc, char *argv[])
diff -up openldap-2.6.10/servers/slapd/overlays/ppolicy.c.13~ openldap-2.6.10/servers/slapd/overlays/ppolicy.c
--- openldap-2.6.10/servers/slapd/overlays/ppolicy.c.13~	2025-05-22 18:56:21.000000000 +0100
+++ openldap-2.6.10/servers/slapd/overlays/ppolicy.c	2026-01-11 20:08:39.919386046 +0000
@@ -33,7 +33,7 @@
 #include "slap.h"
 #ifdef SLAPD_MODULES
 #define LIBLTDL_DLL_IMPORT	/* Win32: don't re-export libltdl's symbols */
-#include <ltdl.h>
+#include <sltdl/sltdl.h>
 #endif
 #include <ac/errno.h>
 #include <ac/time.h>
diff -up openldap-2.6.10/servers/slapd/overlays/remoteauth.c.13~ openldap-2.6.10/servers/slapd/overlays/remoteauth.c
--- openldap-2.6.10/servers/slapd/overlays/remoteauth.c.13~	2025-05-22 18:56:21.000000000 +0100
+++ openldap-2.6.10/servers/slapd/overlays/remoteauth.c	2026-01-11 20:08:39.919386046 +0000
@@ -22,7 +22,7 @@
 #include <ldap.h>
 #if SLAPD_MODULES
 #define LIBLTDL_DLL_IMPORT /* Win32: don't re-export libltdl's symbols */
-#include <ltdl.h>
+#include <sltdl/sltdl.h>
 #endif
 #include <ac/errno.h>
 #include <ac/time.h>
diff -up openldap-2.6.10/servers/slapd/slapi/plugin.c.13~ openldap-2.6.10/servers/slapd/slapi/plugin.c
--- openldap-2.6.10/servers/slapd/slapi/plugin.c.13~	2025-05-22 18:56:21.000000000 +0100
+++ openldap-2.6.10/servers/slapd/slapi/plugin.c	2026-01-11 20:08:39.919386046 +0000
@@ -26,14 +26,14 @@
  * Note: if ltdl.h is not available, slapi should not be compiled
  */
 
-#ifdef HAVE_LTDL_H
+#ifdef HAVE_SLTDL_SLTDL_H
 #include "ldap_pvt_thread.h"
 #include "slap.h"
 #include "slap-config.h"
 #include "slapi.h"
 #include "lutil.h"
 
-#include <ltdl.h>
+#include <sltdl/sltdl.h>
 
 static int slapi_int_load_plugin( Slapi_PBlock *, const char *, const char *, int, 
 	SLAPI_FUNC *, lt_dlhandle *, ConfigArgs *c );
diff -up openldap-2.6.10/build/mod.mk.omv~ openldap-2.6.10/build/mod.mk
--- openldap-2.6.10/build/mod.mk.omv~	2026-01-11 21:07:41.874108854 +0000
+++ openldap-2.6.10/build/mod.mk	2026-01-11 21:08:40.344355293 +0000
@@ -41,7 +41,7 @@ $(LIBRARY): version.lo
 	$(LTLINK_MOD) -module -o $@ $(OBJS) version.lo $(LINK_LIBS)
 
 $(LIBSTAT): version.lo
-	$(AR) ruv $@ `echo $(OBJS) | sed 's/\.lo/.o/g'` version.o
+	$(LIBTOOL) --mode=link $(CC) -o $@ $(OBJS) version.o
 	@$(RANLIB) $@
 
 clean-common: clean-lib FORCE
diff -up openldap-2.6.10/servers/slapd/back-ldif/Makefile.in.omv~ openldap-2.6.10/servers/slapd/back-ldif/Makefile.in
diff -up openldap-2.6.10/build/lib-static.mk.omv~ openldap-2.6.10/build/lib-static.mk
--- openldap-2.6.10/build/lib-static.mk.omv~	2026-01-11 21:42:49.502219379 +0000
+++ openldap-2.6.10/build/lib-static.mk	2026-01-11 21:43:26.122356059 +0000
@@ -17,7 +17,7 @@
 #
 
 $(LIBRARY): version.o
-	$(AR) ru $@ $(OBJS) version.o
+	$(LIBTOOL) --mode=link $(CC) -o $@ $(OBJS) version.o
 	@$(RANLIB) $@
 
 Makefile: $(top_srcdir)/build/lib-static.mk
